Xamarin.Forms.Platform.iOS/Controls/Snackbar/Extensions/UIViewExtensions.cs

```diff
@@ -0,0 +1,93 @@
+﻿using UIKit;
+
+namespace Xamarin.Forms.Platform.iOS.Controls.Snackbar.Extensions
+{
+	/// <summary>
+	/// Provides pre iOS 11 compatible helper properties for for SafeAreaLayoutGuide anchors.
+	/// Helper properties has prefix 'Safe' in their name. 
+	/// If app is running on pre iOS 11, Anchor properties from the view will be used.
+	/// If app is running on iOS 11 or grater, SafeAreaLayoutGuide properties will be used.
+	/// </summary>
+	public static class UIViewExtensions
```

Comment:
The code style is wrong, we use tabs, you can also remove the summary.

I think we should do this internal.